src/org/stringtemplate/v4/compiler/Compiler.java

```diff
@@ -75,7 +75,7 @@
         defaultOptionValues = Collections.unmodifiableMap(map);
     }
 
-    public static Map<String, Short> funcs;
+    public static final Map<String, Short> funcs;
```

Comment:
I think this is an acceptable breaking change, messing with this is unacceptable and never makes sense

Reply:
Well, it doesn't change the fact that people can alter it; they just couldn't reassign it.

Reply:
No, since #228 it's no longer possible to alter it through modification! Now the "loophole" that allowed replacing the entire map is also closed.